src/routes/tools/extreme-precipitation/index.svelte

```diff
@@ -152,7 +153,11 @@
   };
 
   $: $returnPeriodStore, updateIntensity();
-  $: $thresholdTypeStore, $location, $durationStore, updateThreshold();
+  $: $aggregateFnStore,
```

Comment:
More of a tangential question, but I'm wondering if there's any benefit of making the function call reactive this way vs. just passing in the store values as the function params? In the SLR CIS tool I ended up doing the latter (I think) and it works.